src/main/java/me/kodysimpson/simpapi/utility/builders/TextComponentBuilder.java

```diff
@@ -0,0 +1,83 @@
+package me.kodysimpson.simpapi.utility.builders;
+
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.HoverEvent;
+import net.md_5.bungee.api.chat.TextComponent;
+import net.md_5.bungee.api.chat.hover.content.Text;
+import org.bukkit.entity.Player;
+
+/**
+ * Text Component Utility
+ *
+ * @author Negative
+ * June 2021
+ */
+// TODO: TEST
+public class TextComponentBuilder {
```

Comment:
Why does this exist? The BungeeCord chat API already includes [ComponentBuilder](https://javadoc.io/doc/net.md-5/bungeecord-chat/latest/net/md_5/bungee/api/chat/ComponentBuilder.html), which does everything this class does, and more.

Reply:
Not everyone uses Bungeecord

Reply:
Look at the imports in this class.